### DISCUSSION TITLE
Get unexpected warning `<SSRProvider>` from console when upgrade to 11.1.0

### DISCUSSION THREAD

**Discussion by sonck201** [vercel/next.js#28377] [Help]
### What example does this report relate to?

Upgrade nextjs to 11.1.0 and start getting this log from console

### What version of Next.js are you using?

11.1.0

### What version of Node.js are you using?

v16.4.0

### What browser are you using?

Google Chrome - Version 92.0.4515.159 (Official Build) (x86_64)

### What operating system are you using?

maxOS 11.5

### How are you deploying your application?

next dev

### Describe the Bug

Start upgrading to nextjs 11.1.0 and got the log from console
```
When server rendering, you must wrap your application in an <SSRProvider> to ensure consistent ids are generated between the client and server.
When server rendering, you must wrap your application in an <SSRProvider> to ensure consistent ids are generated between the client and server.
When server rendering, you must wrap your application in an <SSRProvider> to ensure consistent ids are generated between the client and server.
```

### Expected Behavior

How to remove this log

### To Reproduce

Start app in DEV mode

**Comment by stefanprobst** [comment]
are you maybe using `react-aria`? looks like the warning is coming from there

**Comment by sonck201** [comment]
> are you maybe using `react-aria`? looks like the warning is coming from there

I'm using `next-intl` only 

Actually, I do not use `react-aria` directly, but I found it in `node_modules`

After following the log, I can skip the log appeared. But I concert that wrapping `<SSRProvider>` may affect the nextjs app

**Reply by lveillard** [reply]
In my case I don't even get the warning removed in my case 🤦‍♀️ when I add the ```<SSRProvider>```

**Comment by dc7290** [comment]
The same log has been confirmed in ver. 12.0.4.

**Comment by Voyager-Two** [comment]
For me, this results in warnings like this (related to [NextUI](https://nextui.org/docs/))
```
react-dom.development.js?8278:67 
       Warning: Prop `id` did not match. Server: "react-aria6362949592-232" Client: "react-aria1563086898-2"
```
![image](https://user-images.githubusercontent.com/3676208/159332498-651cb023-39eb-42c2-a391-f780b1bfe8af.png)

**Comment by lveillard** [comment]
Anyone got a fix for this?

**Comment by marceloverdijk** [comment]
Seeing the same warning while working on a SSG... as in such a case I don't want to use `<SSRProvider>` should I be concerned?

**Comment by mash808** [comment]
I'm not even using `react-aria` as a direct dependency and getting it. Installed react-aria directly just now to use `<SSRProvider>` and it is still happening... Still no fix in the works?
